apps/test/unit/code-studio/pd/professional_learning_landing/landingPageTest.js

```diff
@@ -32,6 +32,7 @@ describe('LandingPage', () => {
     screen.getByTestId('enrolled-workshops-loader');
     expect(screen.queryByText('Online Professional Learning Courses')).to.not
       .exist;
+    screen.getByTestId('action-blocks-wrapper');
```

Comment:
Could this be checked by searching for one of the rendered strings in the static section? Like `i18n.plLandingStaticPLMidHighHeading()`?

Reply:
Oh yeah, good call!